bodhi/server/metadata.py

```diff
@@ -258,8 +258,8 @@ def add_update(self, update):
                     pkg.epoch = '0'
                 pkg.arch = rpm['arch']
 
-                # TODO: how do we handle UpdateSuggestion.logout, etc?
                 pkg.reboot_suggested = update.suggest is UpdateSuggestion.reboot
+                pkg.relogin_suggested = update.suggest is UpdateSuggestion.logout
```

Comment:
Since LGTM complains about the usage of `is` instead of `==`, can you change those two lines?

Otherwise... LGTM!

Reply:
Done in a followup commit!